Makefile

```diff
@@ -96,8 +84,8 @@ crs-instance-id:
 	fi
 	echo "CRS instance ID: $$(kubectl get configmap -n $${BUTTERCUP_NAMESPACE:-crs} crs-instance-id -o jsonpath='{.data.crs-instance-id}')"
 
-deploy-local:
-	@echo "Deploying to local Minikube environment..."
+deploy:
+	@echo "Deploying environment..."
```

Comment:
Line 90 should probably be modified to say something other than Run 'make setup-local' first.'